scripts/configure_motor.py

```diff
@@ -9,21 +9,26 @@
 TARGET_VENDOR_ID = 0x1A86  # e.g., your device's vendor id (in hex)
 TARGET_PRODUCT_ID = 0x55D3  # e.g., your device's product id (in hex)
 
-def find_port(vendor_id, product_id):
+# Update the find_port function to accept a list of (vendor_id, product_id) tuples.
+def find_port(device_ids):
     """
-    Scans available serial ports and returns the port name for the device
-    matching the given vendor_id and product_id.
+    Scans available serial ports and returns the port name for the first device
+    matching any of the given (vendor_id, product_id) pairs.
     """
     ports = list(serial.tools.list_ports.comports())
     for port in ports:
-        # Check if the port has the specified vendor_id and product_id
-        #print(f"Found port: {port.device}, VID: {hex(port.vid) if port.vid is not None else 'None'}, PID: {hex(port.pid) if port.pid is not None else 'None'}")
-        # port.vid and port.pid are provided by pyserial if available
-        if port.vid == vendor_id and port.pid == product_id:
-            print(f"Found device on port: {port.device}")
-            return port.device
+        for vendor_id, product_id in device_ids:
+            if port.vid == vendor_id and port.pid == product_id:
+                print(f"Found device on port: {port.device} (VID: {hex(vendor_id)}, PID: {hex(product_id)})")
+                return port.device
     return None
 
+# Update the script to use a list of device IDs.
+DEVICE_IDS = [
```

Comment:
To this table, this is what the second commit is for, so that if the IDs are different you can add as many IDs to this table.